tests/unit_tests/test_helm_chart.py

```diff
@@ -256,11 +238,13 @@ def test_do_not_have_to_provide_scratch_host_path_twice():
     config = yaml.safe_load(
         manifests["ConfigMap"]["blueapi-config"]["data"]["config.yaml"]
     )
-    init_config = yaml.safe_load(
-        manifests["ConfigMap"]["blueapi-initconfig"]["data"]["initconfig.yaml"]
-    )
+    init_config = manifests["ConfigMap"]["blueapi-initconfig"]["data"][
```

Comment:
Somewhere in the initial safe_load of the grouped manifest, the initContainer config and the main containerConfig are being found to be the same object and being cached. I'm not certain why- we could have even have assert config["scratch"] is init_config["scratch"] below, they're the same object, so trying to yaml_Load it a second time as it was doing was breaking..
I'm going to replicate this comment on the PR into main